tensorboard/webapp/plugins/npmi/views/annotations_list/annotations_list_header/annotations_list_header_component.scss

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+.annotations-header-container {
+  display: flex;
+  flex-wrap: nowrap;
+  font-size: 0.9em;
+  font-weight: 500;
+  padding-top: 10px;
+  width: calc(100% - 100px);
```

Comment:
Ideally you don't have to specify width value dependent on width values of other elements (I assume the 100px here is the 90px width of the checkbox + 10px of its margin-left).

I think instead you want `flex-grow: 1` which effectively states that this element should take up the remainder of the flex box width.